src/main/java/decodes/cwms/resevapcalc/ResEvap.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (c) 2018
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
```

Comment:
@MikeNeilson -- do we need to update this type of comment 'USACE PROPRIETARY/CONFIDENTIAL.'    ?

Reply:
Yeah.

```
/*
* Where Applicable, Copyright 2024 The OpenDCS Consortium or it's contributors.
* 
* Licensed under the Apache License, Version 2.0 (the "License"); you may not
* use this file except in compliance with the License. You may obtain a copy
* of the License at
* 
*   http://www.apache.org/licenses/LICENSE-2.0
* 
* Unless required by applicable law or agreed to in writing, software 
* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
* License for the specific language governing permissions and limitations 
* under the License.
*/
```